src/view/com/composer/ComposerReplyTo.tsx

```diff
@@ -100,7 +110,8 @@ export function ComposerReplyTo({replyTo}: {replyTo: ComposerOptsPostRef}) {
             <Text
               type="post-text"
               style={pal.text}
-              numberOfLines={!showFull ? 6 : undefined}>
+              numberOfLines={!showFull ? 6 : undefined}
+              selectable>
```

Comment:
Should it be a `selectable uitextview`? Would that cause issues?

Reply:
This `<Text>` does that underneath in https://github.com/bluesky-social/social-app/blob/67fca733faafdc54f85846ca0f0bbf0f4f174d13/src/view/com/util/text/Text.tsx#L35

(Reminds me though that we can also merge this in https://github.com/bluesky-social/social-app/pull/3090, has a few fixes that came up from other people using it)

Reply:
Ah my bad, should've checked that :)